.storybook/main.js

```diff
@@ -12,8 +17,26 @@ module.exports = {
     }
   ],
   webpackFinal: config => {
+    config.resolve.plugins = config.resolve.plugins || [];
     config.resolve.plugins.push(new TsconfigPathsPlugin());
 
+    config.resolve.fallback = {
+      "stream": false,
+      "path": false,
+      "process": false
+    };
+
+    config.module.rules.push({
+      test: /\.m?js/,
+      resolve: {
+        fullySpecified: false
+      }
+    });
+
+    config.plugins.push(new ProvidePlugin({
+      process: require.resolve('process/browser'),
```

Comment:
Why is it needed to provide `process` here?

Reply:
Well, unfortunately way too many libraries we depend on are not really browser-compatible, mainly they use some node internals, in this case usually the `process.env.*` global. While we can try to eliminate this in the future, this plugin provides a `process` global so the code works for the time being. 

AFAIK we have the same provide plugin in the gatsby config (that's where I took the idea from)